src/lang/en-US.json

```diff
@@ -171,8 +171,6 @@
   "StepNavigation.controls.next.label": "Next",
   "StepNavigation.controls.prev.label": "Prev",
   "StepNavigation.controls.finish.label": "Finish",
-  "Admin.TaskAnalysisTable.taskCountStatus": "Shown: {countShown} Tasks",
```

Comment:
These modified message ids need to also be manually modified in the translation files in `src/lang/` or else those translations will no longer be picked up for those messages. It looks `ja.json` and `ko.json` both contain active translations for these.

Reply:
"Admin.TaskAnalysisTableHeader.taskCountStatus" was moved further down the file -- so it is still being used as that id.